### PR TITLE
Feat playwrigt actions assertions

### PR DESCRIPTION
**Pull request by TaniaBystrova**

### CODE REVIEW

File: tests/auth/register.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ await lastNameInput.fill(' ')
+ await emailInput.fill(' ')
+ await passwordInput.fill(' ')
+ await repeatPasswordInput.fill(' ')
+ await registerButton.scrollIntoViewIfNeeded();
+ await expect(registerButton).toBeDisabled()
+ await page.waitForSelector('.invalid-feedback')
+ const validationMessages = await page.$$eval('.invalid-feedback', (elements) =>
+    elements.map((element) => element.textContent.trim())
+  )
```
Comment (sttaran) [inline tests/auth/register.spec.js]: https://playwright.dev/docs/api/class-locator#locator-all-inner-texts

File: tests/auth/register.spec.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ await expect(registrationTitle).toHaveText('Registration')
+ await nameInput.fill(' ')
+ //await expect(registrationPopup).toHaveInne('p', {containsText: 'Name is invalid'})
+ await lastNameInput.fill(' ')
+ await emailInput.fill(' ')
+ await passwordInput.fill(' ')
+ await repeatPasswordInput.fill(' ')
+ await registerButton.scrollIntoViewIfNeeded();
+ await expect(registerButton).toBeDisabled()
+ await page.waitForSelector('.invalid-feedback')
```
Comment (sttaran) [inline tests/auth/register.spec.js]: create locator for this elements

File: tests/auth/register.spec.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+
+ 
+ await signInButton.click()
+ await expect(loginPopup).toBeVisible()
+ await registrationButton.click()
+ await expect(registrationPopup).toBeVisible()
+ await expect(registrationTitle).toHaveText('Registration')
+ await nameInput.fill(' ')
+ //await expect(registrationPopup).toHaveInne('p', {containsText: 'Name is invalid'})
```
Comment (sttaran) [inline tests/auth/register.spec.js]: remove unused code

File: tests/auth/register.spec.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    'Last name is invalidLast name has to be from 2 to 20 characters long',
+    'Email is incorrect',
+    'Password has to be from 8 to 15 characters long and contain at least one integer, one capital, and one small letter',
+  ];
+
+
+ 
+ await signInButton.click()
+ await expect(loginPopup).toBeVisible()
+ await registrationButton.click()
```
Comment (sttaran) [inline tests/auth/register.spec.js]: open popup by click on this element
image

File: tests/auth/register.spec.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+   await nameInput.fill('John');
+    await lastNameInput.fill('Doe');
+    await emailInput.fill(`hillel-${userEmail}`);
+    await passwordInput.fill('StrongP@ss123');
+    await repeatPasswordInput.fill('StrongP@ss123');
+
+    // Scroll to the "Register" button and click it
+    await registerButton.scrollIntoViewIfNeeded();
+    await registerButton.click();
+    expect(page).toHaveURL('/panel/garage')
```
Comment (sttaran) [inline tests/auth/register.spec.js]: add assertion that registration popup is not visible anymore

**Review by sttaran — CHANGES_REQUESTED** [review]

File: tests/auth/register.spec.js (hunk 1 above)
Comment (TaniaBystrova) [reply tests/auth/register.spec.js]: +

File: tests/auth/register.spec.js (hunk 2 above)
Comment (TaniaBystrova) [reply tests/auth/register.spec.js]: +

File: tests/auth/register.spec.js (hunk 3 above)
Comment (TaniaBystrova) [reply tests/auth/register.spec.js]: +

File: tests/auth/register.spec.js (hunk 4 above)
Comment (TaniaBystrova) [reply tests/auth/register.spec.js]: +

File: tests/auth/register.spec.js (hunk 5 above)
Comment (TaniaBystrova) [reply tests/auth/register.spec.js]: +

**Review by sttaran — APPROVED** [review]

**Pull request merged**